VisualizationBase/styles/item/VViewItemNode/default

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE EnvisionVisualizationStyle>
+<style prototypes="../Item/default">
```

Comment:
Since VViewItemNode inherits from declarativeBase, the correct thing here is to use "../DeclarativeItemBase/default" as a prototype. This will also make the next two lines unnecessary (close the style tag directly on this line).

I will merge the code as it is now, but please fix this in the next pull request.